src/main/java/sg/ncl/MainController.java

```diff
@@ -3404,6 +3444,7 @@ public String approveTeam(
             log.warn("Approve team {} FAIL", teamId);
             redirectAttributes.addFlashAttribute(MESSAGE, ERR_SERVER_OVERLOAD);
         }
+
         return "redirect:/admin";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Approve team request fail" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)